validation.go

```diff
@@ -47,12 +53,14 @@ type validator struct {
 	hash       HashFunc
 }
 
-func (v *validator) calcRoot(stopAtLayer uint) ([]byte, error) {
+func (v *validator) calcRoot(stopAtLayer uint) (root []byte, parkingSnapshots [][][]byte, err error) {
```

Comment:
In general, the main reason I named the return values was documentation. Since I didn't use naked `return` statements, shadowing is not an issue, imo.

Anyways, since I've now created a type for `ParkingSnapshot` the return values are clearer now even without the names and I don't mind removing the names, so I've done that.